isthmus/src/test/java/io/substrait/isthmus/ProtoRelConverterTest.java

```diff
@@ -0,0 +1,82 @@
+package io.substrait.isthmus;
+
+import io.substrait.expression.proto.FunctionLookup;
+import io.substrait.function.SimpleExtension;
+import io.substrait.proto.Plan;
+import io.substrait.proto.PlanRel;
+import io.substrait.relation.ProtoRelConverter;
+import io.substrait.relation.Rel;
+import org.apache.calcite.sql.parser.SqlParseException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class ProtoRelConverterTest extends PlanTestBase {
```

Comment:
It looks like this tests don't verify any of the versions, just that we don't throw. Is that correct? Maybe add a comment to that effect throughout with a todo?

Reply:
The tests verify that a proto is convertable to a pojo and that when the pojo is converted back to a proto it is equivalent to the original proto.

Reply:
Can you point out the assertions? I don't see them.